api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/MakePaymentMessageProcessor.java

```diff
@@ -30,18 +36,44 @@
 
     private PaymentService paymentService;
     private MessageContextLoader messageContextLoader;
+    private PaymentCycleEntitlementCalculator paymentCycleEntitlementCalculator;
     private MessageQueueClient messageQueueClient;
+    private ChildDateOfBirthCalculator childDateOfBirthCalculator;
 
     @Override
     @Transactional(Transactional.TxType.REQUIRES_NEW)
     public MessageStatus processMessage(Message message) {
         MakePaymentMessageContext messageContext = messageContextLoader.loadMakePaymentContext(message);
-        paymentService.makePaymentForCycle(messageContext.getPaymentCycle(), messageContext.getCardAccountId());
-        EmailMessagePayload messagePayload = buildPaymentNotificationEmailPayload(messageContext.getPaymentCycle());
+        PaymentCycle paymentCycle = messageContext.getPaymentCycle();
+        paymentService.makePaymentForCycle(paymentCycle, messageContext.getCardAccountId());
+        EmailMessagePayload messagePayload = buildPaymentNotificationEmailPayload(paymentCycle);
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
+        int numChildrenTurningFourInNextMonth = childDateOfBirthCalculator.getNumberOfChildrenTurningFourAffectingNextPayment(paymentCycle);
```

Comment:
It's fine for now, but I'm pretty sure we're going to find we want to move the email decisions and generation into a separate component

Reply:
Yeah, that's pretty obvious as this will grow but we'll leave it for now.